src/style.css

```diff
@@ -1 +1,91 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 20px 8px 0;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). The `.container` class has both vertical and horizontal margins. Consider using only top or bottom margins for consistency and to avoid potential margin collapse issues.